src/modules/functions.js

```diff
@@ -323,6 +323,328 @@ export const displayUserNameOverlay = () => {
   playerHeaderTarget.insertAdjacentElement("beforebegin", userOverlayContainer);
 };
 
+export const toggleTokenConversion = (toggle) => {
+  const tokenToUsdRate = 0.0828; // Conversion rate from tokens to USD
+  const cfg = config.get();
```

Comment:
import the value from constants after you move it there - no need to store all settings in memory again